bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -595,14 +595,18 @@ private string GetName(IEnumerable<Claim> claims, IEnumerable<string> additional
             return null;
         }
 
-        private async Task DeleteExistingSsoUserRecord(Guid userId, Guid orgId, OrganizationUser orgUser)
+        private async Task CheckForExistingSsoUserRecord(Guid userId, Guid orgId, OrganizationUser orgUser)
```

Comment:
Not super happy about the naming of this method. Is there any benefit over logging the event here vs in ` CreateSsoUserRecord`?

Reply:
I've moved this to the main method as discussed